### PR TITLE
Now we store the metadata in setup.cfg.

### PR DESCRIPTION
**Pull request by KOLANICH**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,47 +1,13 @@
...
 from kaitaistruct import __version__
 
-with open('README.rst', encoding='utf-8') as f:
-    long_description = f.read()
+this_dir=os.path.dirname(__file__)
+cfg = read_configuration(os.path.join(this_dir, 'setup.cfg'))
+cfg["metadata"]["version"]=__version__
+#print(cfg)
+cfg["options"].update(cfg["metadata"])
+cfg=cfg["options"]
```
Comment (GreyCat) [inline setup.py]: Please fix this broken formatting. Literally all style checkers (pep8, pycodestyle, pylint) complain about:

```
C:  6, 0: Exactly one space required around assignment
C:  8, 0: Exactly one space required around assignment
C: 11, 0: Exactly one space required around assignment
```

Probably optional:

```
C:  6, 0: Constant name "this_dir" doesn't conform to UPPER_CASE naming style (invalid-name)
C:  7, 0: Constant name "cfg" doesn't conform to UPPER_CASE naming style (invalid-name)
C: 11, 0: Constant name "cfg" doesn't conform to UPPER_CASE naming style (invalid-name)
C:  3, 0: standard import "import os" should be placed before "from setuptools import setup" (wrong-import-order)
```

File: setup.cfg [hunk 2]
```diff
@@ -1,5 +1,39 @@
+[metadata]
+name = kaitaistruct
+version = 0.1.0
```
Comment (GreyCat) [inline setup.cfg]: Shall we have it here in the first place, given that we override it in the code? If it's mandatory, could we put something clearly invalid (like `???` or `xxx` or `0.0.0`) here?

**Comment by GreyCat** [comment]
Thanks for this contribution! Could you fix these style issues, so I can merge it?

File: setup.cfg (hunk 2 above)
Comment (KOLANICH) [reply setup.cfg]: Oops, I have forgotten to remove it after I have copied my boilerplate.

**Comment by GreyCat** [comment]
Looks legit! Thanks!

**Pull request merged**